libs/blocks/merch-card/merch-card.js

```diff
@@ -228,6 +228,24 @@ const simplifyHrs = (el) => {
   });
 };
 
+function createQuantitySelect(el) {
+  const quantitySelectConfig = el.querySelector('ul');
```

Comment:
I don't know if it's a good idea to hijack all `<ul>` tags of the entire block for this. I have seen other merch card variants coming up soon with unordered list text. Can we instead be more specific to detect quantity select? How about creating a new row that starts with `<ul>` for this?

Thoughts? @3ch023 @yesil @npeltier @honstar

Reply:
Yes, the selector should be more specific.
e.g:

get the latest `li` of a `ul`, and  check that it starts with a numerical value, if yes, assume that its parent `ul` is the one you are looking for.

Reply:
In my case, I added the check for card-type plans, and in step 2, I'm checking if it contains 'Quantity' inside, so it's already pretty specific. 
<img width="318" alt="Screenshot 2024-01-16 at 4 33 44 PM" src="https://github.com/adobecom/milo/assets/19995557/09a4dc20-6cc2-418d-b36d-85515334bf1e">
<img width="508" alt="Screenshot 2024-01-16 at 4 34 28 PM" src="https://github.com/adobecom/milo/assets/19995557/900e84d5-4e46-4871-89b9-8451f94fe75f">